app/models/diffusion_history.rb

```diff
@@ -1,6 +1,13 @@
 class DiffusionHistory < ApplicationRecord
   belongs_to :practice
-  belongs_to :va_facility
+  belongs_to :va_facility, optional: true
+  belongs_to :clinical_resource_hub, optional: true
+
+
+  #TODO
```

Comment:
i think we can remove this TODO since we are validating it

Reply:
This has been removed.